web-marketplace/src/pages/Dashboard/Dashboard.constants.tsx

```diff
@@ -11,6 +11,8 @@ import { ProjectPageIcon } from 'web-components/src/components/icons/ProjectPage
 const className = 'pl-20';
 const labelClassName = 'text-sm';
 
+export const DISABLED_DROPDOWN_ITEM_TOOLTIP_TEXT = msg`This is not yet supported by Wallet Connect and Keplr mobile. Please log in using Keplr browser extension on desktop`;
```

Comment:
This is the exact same message in the existing `NOT_SUPPORTED_TOOLTIP_TEXT`, except for the dot that is missing at the end. Can we use the same everywhere?